src/test/ui/rustdoc/issue-83512.rs

```diff
@@ -0,0 +1,10 @@
+// Regression test for the ICE described in #83512.
```

Comment:
Let's add this test in `src/test/ui/rustdoc/doc_keyword.rs` instead

Reply:
Fixed. Thanks for reviewing this PR!